jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaResourceSelector.kt

```diff
@@ -14,26 +14,31 @@ import software.aws.toolkits.jetbrains.ui.ResourceSelector
 import javax.swing.JComponent
 
 class SchemaResourceSelector {
-    private var currentAwsConnection: ConnectionSettings? = null
+    var awsConnection: ConnectionSettings? = null
+        set(value) {
+            val previous = field
+            field = value
+            // if we are loading for the first time (creation), do not load the schemas selector yet
+            if (previous != null) {
+                schemasSelector.reload()
+            }
+        }
```

Comment:
`Delegates.observable` can simplify this

Reply:
Actually I dont think we need to make any changes to this class. Changing the components to be lazy in the other file should be enough

Reply:
Slightly different, I guess that works. We need to edit this class or it loads the panel when the item is created as we already reload when this is changed